src/controllers/company.number.controller.ts

```diff
@@ -0,0 +1,42 @@
+import { Session } from "ch-node-session-handler";
+import { NextFunction, Request, Response } from "express";
+import { ObjectionCompanyProfile } from "../model/objection.company.profile";
+import { OBJECTIONS_CONFIRM_COMPANY } from "../model/page.urls";
+import { getCompanyProfile } from "../services/company.profile.service";
+import { addToObjectionsSession, createObjectionsSession } from "../services/objections.session.service";
+import logger from "../utils/logger";
+
+/**
+ * POST controller for company number screen
+ * @param req
+ * @param res
+ * @param next
+ */
+const route = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    logger.debug("Attempt to retrieve and show the company details");
```

Comment:
as this is logging that we've entered the method, you can use debugRequest which will also log out the url path
eg. logger.debugRequest(req, "Attempt to retrieve and show the company details");

Reply:
updated 2 logging methods with this.